### DISCUSSION TITLE
Slow graph response

### DISCUSSION THREAD

**Discussion by Predatoroid** [OpenBB-finance/OpenBB#1321] [Q&A]
Hi,

I just installed gamestonk terminal (using docker) & also the VcXsrv. When I run e.g.:
> stocks
> load FB
> ta
> macd

The graph is shown successfully but the response is lagging after zooming or dragging the chart.
Do you have any idea what this is about?

Thanks in advance !

**Comment by JeamsH** [comment]
Playing different types of games is always interesting. I love to play them.

**Comment by DidierRLopes** [comment]
Hey @Predatoroid ,

Does this still occur?
